client/layout/masterbar/oauth-client.scss

```diff
@@ -121,6 +121,7 @@
 
 	.card {
 		--color-link-hover: #4ccee4;
+		clip-path: unset;
```

Comment:
`.card` has a clip-path set, which cuts of the focus ring in some instances, this being Crowdsignal